src/doc/CMakeLists.txt

```diff
@@ -66,4 +66,4 @@ add_custom_target(gui_manual ALL
         -a -q -b html
         "${CMAKE_CURRENT_SOURCE_DIR}"
         "${SPHINX_HTML_DIR}"
-    COMMENT "Building Sphinx-HTML documentation for GUI")
+    COMMENT "Building Sphinx-HTML documentation for GUI and CLI")
```

Comment:
Since this is now building both gui and cli manual, perhaps the custom target name should be changed from 'gui_manual' to perhaps 'user_manuals'.

Reply:
Done